new/LIT6392HymnInnocents.xml

```diff
@@ -0,0 +1,79 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6392HymnInnocents" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሕፃናት፡ ኄራን፡ ወንጹሐን፡ እምርኵስ፡ ናሁ፡ ወጠንኩ፡ ሰላመክሙ፡ በብርሃነ፡ ጥበብ፡ ቅዱስ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Ḥǝḍānāt ḫerān wä-neṣuḥan ʾǝmerǝkwus nāhu wäṭänku sälāmäkemu bä-bǝrhanä ṭǝbäb qǝddus</title>
```

Comment:
Please correct the transliteration. The opposition long and short "a"; ʾǝmerǝkwus > ʾǝm-rǝkwus, as far as I understand, and labialized k with raise w (see https://betamasaheft.eu/Guidelines/?id=transliteration-principles).